index.html

```diff
@@ -11,6 +11,18 @@
     <title>Juan Portfolio</title>
 </head>
 <body>
+
+    <header class="mobile-menu no-visible" id="mobile_menu">
+        <nav>
+            <div id="close_menu_icon">X</div>
+            <ul id="mobile_link_list">
+                <li> <a href="#">Portfolio</a> </li>
+                <li> <a href="#">About</a> </li>
+                <li> <a href="#">Contact</a> </li>
```

Comment:
- [ ] Kindly make sure when one of the links is clicked we directly move to the corresponding part of the page. If the `Contact` link is clicked, we move directly to the contact part of your portfolio.